cms/management/commands/create_initial_data.py

```diff
@@ -10,9 +10,21 @@ class Command(BaseCommand):
 
     help = 'Create initial data by using factories.'
 
-    def collect_initial_data_functions(self):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--app_label',
```

Comment:
Nit: app_label -> app-label (or we can just use what Django uses, I don't remember)